src/utils/streams.js

```diff
@@ -0,0 +1,158 @@
+import program from "commander"
+import readline from "readline"
+import reverseString from "reverse-string"
+import fs from "fs"
+import csv from "csvtojson"
+import Deque from "collections/deque";
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+function reverse() {
+  rl.question("Please provide a string for reversing: ", (answer) => {
+    console.log(`Reversed: ${reverseString(answer)}`);
```

Comment:
Remove `reverseString` package. There's a simple solution in JS
```
answer.split('').reverse().join('');
```

Reply:
it's not so straight-forward and clear actually, but yes, pretty simple =) will do.